docs/how-to/analytics-dashboard.md

```diff
@@ -0,0 +1,54 @@
+---
+title: Analytics Dashboard
+sidebar_label: Analytics Dashboard
+slug: /analytics-dashboard
+---
+
+# Analytics Dashboard
+
+Amplication's Analytics Dashboard is a powerful tool that provides developers with valuable insights into their application's development process.
+
+The dashboard highlights key metrics related to productivity, efficiency, and code quality to help you understand the impact of using Amplication on your development workflow and the overall return on investment.
+
+To start leveraging the insights provided by the Analytics Dashboard, navigate to the Workspace Overview or Project Overview screen in your Amplication account.
+
+## Productivity Metrics
+
+The Productivity Metrics section provides an overview of the development activity and code generation. This lets developers track their progress and identify trends in their development process using the **Overtime Graph** and **Generated Lines Of Code** sections.
+
+### Overtime Graph
+
+The Overtime Graph is a visual representation of the development activity over a specified period. It displays the number of builds, entities, APIs, and plugins created or updated, allowing developers to track their progress and identify trends in their development process.
```

Comment:
There is a difference between the number of builds and the rest (entities, APIs, and plugins). The # builds is easy- how many builds were generated during the time. The others is about how many "objects" were created or changed (updated). Let's better phrase this sentence to reflect it, so it will be clear what we are counting here.

Reply:
Another thing- "allowing developers to track their progress and identify trends in their development process." - Sounds like a repetitive sentence that was mentioned before. Let's use different words or such